src/ripple/app/ledger/MetaView.h

```diff
@@ -150,41 +238,79 @@ class MetaView : public View
     void
     update (std::shared_ptr<SLE> const& sle) override;
 
-    bool
-    openLedger() const override;
-
     //--------------------------------------------------------------------------
 
-    /** Apply changes to the parent View */
+    /** Apply changes to the base View.
+
+        `to` must contain contents identical to the
+        parent view passed upon construction, else
+        undefined behavior will result.
+
+        After a call to apply(), the only valid operation
+        on the object is a call to the destructor.
+    */
     void
-    apply();
+    apply (BasicView& to,
+        beast::Journal j = {});
+
```

Comment:
Yes.  It colloquially known as "rvalue *this".  It applies the rvalue reference qualifier (or lvalue reference qualifier) to `*this` in the same way that a `const` member function applies `const` to `*this`.